t/Renard/Curie/Component/KeyBindings.t

```diff
@@ -0,0 +1,66 @@
+use Test::Most;
+use Test::More;
```

Comment:
FYI:

`Test::Most` pulls in `Test::More` and a bunch of other useful testing modules. That way all the imports get out of your way and you only need to import `Test::Most`.